cmd/atlas/atlas.go

```diff
@@ -53,9 +55,73 @@ func initConfig() {
 	}
 }
 
+// createConfigFromMongoCLIConfig creates the atlasCLI config file from the mongocli.toml.
+func createConfigFromMongoCLIConfig() {
+	atlasConfigHomePath, err := config.AtlasCLIConfigHome()
+	if err != nil {
+		return
+	}
+
+	atlasConfigPath := fmt.Sprintf("%s/%s", atlasConfigHomePath, "config.toml")
+	f, err := os.Open(atlasConfigPath) // if config.toml is already there, exit
+	if err == nil {
+		return
+	}
+	defer f.Close()
+
+	mongoCLIConfigPath := mongoCLIConfigPath()
+	if mongoCLIConfigPath == "" {
+		return
+	}
+
+	in, err := os.Open(mongoCLIConfigPath)
+	if err != nil {
+		return
+	}
+	defer in.Close()
+
+	_, err = os.Stat(atlasConfigHomePath) // check if the dir is already there
+	if err != nil {
+		defaultPermissions := 0700
+		if err = os.Mkdir(atlasConfigHomePath, os.FileMode(defaultPermissions)); err != nil {
+			return
+		}
+	}
+
+	out, err := os.Create(atlasConfigPath)
+	if err != nil {
+		return
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, in)
+	if err != nil {
+		log.Printf("we cannot create %s from your %s: %v", atlasConfigPath, mongoCLIConfigPath, err)
+	}
+
+	_, _ = fmt.Fprintf(os.Stderr, `we have used %s to generate %s
+
+`, mongoCLIConfigPath, atlasConfigPath)
```

Comment:
I think We can delete this, or change it to be more generic 

> Atlas CLI has copied your MongoCLI config to: %s

Reply:
Updated